pkg/webhook/webhook.go

```diff
@@ -398,6 +405,76 @@ func (w *Config) buildFleetValidatingWebhooks() []admv1.ValidatingWebhook {
 			},
 			TimeoutSeconds: longWebhookTimeout,
 		},
+		{
+			Name:                    "fleet.managedresource.validating",
+			ClientConfig:            w.createClientConfig(managedresource.ValidationPath),
+			FailurePolicy:           &failFailurePolicy,
+			SideEffects:             &sideEffortsNone,
+			AdmissionReviewVersions: admissionReviewVersions,
+			Rules: []admv1.RuleWithOperations{
+				{
+					Operations: []admv1.OperationType{
+						admv1.Create, admv1.Update, admv1.Delete,
+					},
+					Rule: createRule(
+						[]string{
+							placementv1beta1.GroupVersion.Group,
+						},
+						[]string{
+							placementv1beta1.GroupVersion.Version,
+						},
+						[]string{
+							placementv1beta1.ClusterResourcePlacementResource,
+						}, &clusterScope),
+				},
+				{
+					Operations: []admv1.OperationType{
+						admv1.Create, admv1.Update, admv1.Delete,
+					},
+					Rule: createRule(
```

Comment:
nit: Since we separated the rules, I guess we can have the createRule call in one line, make it concise